gap/perm.gi

```diff
@@ -130,7 +134,13 @@ FindHomMethodsPerm.PcgsForBlocks := function(ri,G)
 end;
 
 #! @BeginChunk BalTreeForBlocks
-#! TODO
+#! This method creates a balanced composition tree for the kernel of an
+#! imprimitive group. This is guaranteed as
+#! <C>FindHomMethodsPerm.BalTreeForBlocks</C> is
+#! just called from <C>FindHomMethodsPerm.Imprimitive</C> and itself.
```

Comment:
I would recommend to use proper refs here, something like `<Ref Subsect="Imprimitive"/>` or so should do it.